push-image-ghcr/src/metadata.ts

```diff
@@ -0,0 +1,35 @@
+import { getExecOutput } from '@actions/exec'
+import { parse as semverParse } from 'semver'
+import { getRef, getSha, getTag } from '../../lib/github'
+
+export async function setTags() {
```

Comment:
get verb matches logic better

```suggestion
export async function getTags() {
```

Reply:
done